app/controllers/inventories_controller.rb

```diff
@@ -0,0 +1,42 @@
+class InventoriesController < ApplicationController
+  def index
+    @user = current_user
+    @inventories = @user.inventories
+  end
+
+  def show
+    @user = current_user
+    @inventory = Inventory.find(params[:id])
+    @inventory_foods = @inventory.inventory_foods
```

Comment:
- [ ] Since we are getting the inventory food name in the `app/views/show.html.erb` view at line 24 it would be nice to include **food** when getting `inventory_foods` of an inventory to avoid having the **N+1** query problem